pkg/scheduler/partition.go

```diff
@@ -443,6 +444,13 @@ func (pc *PartitionContext) getApplication(appID string) *objects.Application {
 	return pc.applications[appID]
 }
 
+func (pc *PartitionContext) getRejectedApplication(appID string) *objects.Application {
+	pc.RLock()
+	defer pc.RUnlock()
+
+	return pc.rejectedApplications[appID]
+}
+
```

Comment:
This is just used once in a test, in the line above in the test we directly check the map. Accessing the map in the test directly should be OK.